File: Bossinfo.Module.Hospital/orderregresult.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Bossinfo.Module.Hospital
{
    public class orderregresult
    {
        [Key]
        [Column(Order = 0)]
        [DatabaseGenerated(DatabaseGeneratedOption.None)]
        public long ID { get; set; }

        [Key]
        [Column(Order = 1)]
        [DatabaseGenerated(DatabaseGeneratedOption.None)]
        public long HospitalID { get; set; }

        [Key]
        [Column(Order = 2)]
        [StringLength(10)]
        public string InspectionNumber { get; set; }

        [StringLength(10)]
        public string Idno { get; set; }

        [StringLength(40)]
        public string Name { get; set; }

        [Column(TypeName = "datetime2")]
        public DateTime? InspectionDateTime { get; set; }

        [Column(TypeName = "datetime2")]
        public DateTime? SendInspectionDateTime { get; set; }

        [Column(TypeName = "datetime2")]
        public DateTime? RegDateTime { get; set; }

        [StringLength(20)]
        public string RegNo { get; set; }

        [Key]
        [Column(Order = 3)]
        public bool IsDelete { get; set; }

        [Column(TypeName = "datetime2")]
        public DateTime? CreatedTime { get; set; }

        public long? CreatedBy { get; set; }

        [Column(TypeName = "datetime2")]
        public DateTime? LastModifiedTime { get; set; }

        public long? LastModifiedBy { get; set; }
    }
}

File: Bossinfo.Models.Hospital/orderregresult.ts

 //改NameSpace就好
 
    import { BossApiModel, Key } from '@boss/decorator/BossApiModel';
    import { I18nNamespace } from '@boss/decorator/i18n/i18nNs';
    @BossApiModel('api/orderregresult')
    @I18nNamespace('Model.EDI.orderregresult')
    export class orderregresult {
        
        @Key()// Column(Order = 0)// DatabaseGenerated(System.ComponentModel.DataAnnotations.Schema.DatabaseGeneratedOption.None)
        public ID: number;
        @Key()// Column(Order = 1)// DatabaseGenerated(System.ComponentModel.DataAnnotations.Schema.DatabaseGeneratedOption.None)
        public HospitalID: number;
        @Key()// Column(Order = 2)// StringLength(10)
        public InspectionNumber: string;
        // StringLength(10)
        public Idno: string;
        // StringLength(40)
        public Name: string;
        // Column(TypeName = "datetime2")
        public InspectionDateTime: Date;
        // Column(TypeName = "datetime2")
        public SendInspectionDateTime: Date;
        // Column(TypeName = "datetime2")
        public RegDateTime: Date;
        // StringLength(20)
        public RegNo: string;
        @Key()// Column(Order = 3)
        public IsDelete: boolean;
        // Column(TypeName = "datetime2")
        public CreatedTime: Date;
        
        public CreatedBy: number;
        // Column(TypeName = "datetime2")
        public LastModifiedTime: Date;
        
        public LastModifiedBy: number;
    }
    
    /*
    "orderregresult": {
        
        "ID": "ID",
        "HospitalID": "HospitalID",
        "InspectionNumber": "InspectionNumber",
        "Idno": "Idno",
        "Name": "Name",
        "InspectionDateTime": "InspectionDateTime",
        "SendInspectionDateTime": "SendInspectionDateTime",
        "RegDateTime": "RegDateTime",
        "RegNo": "RegNo",
        "IsDelete": "IsDelete",
        "CreatedTime": "CreatedTime",
        "CreatedBy": "CreatedBy",
        "LastModifiedTime": "LastModifiedTime",
        "LastModifiedBy": "LastModifiedBy"
    }
    */
    
